admin/sql/CreateTriggers.sql

```diff
@@ -52,6 +52,9 @@ CREATE TRIGGER del_collection_sub_on_delete BEFORE DELETE ON editor_collection
 CREATE TRIGGER del_collection_sub_on_private BEFORE UPDATE ON editor_collection
     FOR EACH ROW EXECUTE PROCEDURE del_collection_sub_on_private();
 
+CREATE TRIGGER restore_collection_sub_on_public BEFORE UPDATE ON editor_collection
+    FOR EACH ROW EXECUTE PROCEDURE restore_collection_sub_on_public();
```

Comment:
I think `AFTER` makes more sense here, since we're propagating the changes to another table and want to be sure we have the final version of the row.
```suggestion
CREATE TRIGGER restore_collection_sub_on_public AFTER UPDATE ON editor_collection
    FOR EACH ROW EXECUTE PROCEDURE restore_collection_sub_on_public();
```